messages_appendix.cddl

```diff
@@ -422,16 +426,16 @@ text-track-mode = &(
 
 text-track-cue = {
   1: text ; id
-  2: media-time-range ; range
+  2: media-timeline-range ; range
   3: text ; text
 }
 
-media-time-range = [
-  start: media-time,
-  end: media-time
+media-timeline-range = [
```

Comment:
Nit: consider declaring above next to `media-timeline`, especially since it is used above.

Reply:
Done